pkg/agent/apiserver/handlers/getpodinterface/handler.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package getpodinterface
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+// GetPodInterfaceResponse describes the internal response struct of the pod command.
+type GetPodInterfaceResponse struct {
+	PodName         string                            `json:"name" yaml:"name" antctl:"key,The name of the pod"`
+	InterfaceConfig []*interfacestore.InterfaceConfig `json:",inline" yaml:",inline"`
+}
+
+// HandleFunc returns the function which can handle queries issued by the pod command,
+func HandleFunc(aq monitor.AgentQuerier) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if aq == nil {
+			w.WriteHeader(http.StatusNotImplemented)
+			return
+		}
+
+		ns, _ := r.URL.Query()["namespace"]
+
+		key, doFilter := r.URL.Query()["name"]
+		if doFilter && (len(key) != 1 || len(key[0]) == 0) {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		m := make(map[string][]*interfacestore.InterfaceConfig)
+		for _, v := range aq.GetInterfacesByType(interfacestore.ContainerInterface) {
+			podName := (*v.ContainerInterfaceConfig).PodName
+			vPodNS := (*v.ContainerInterfaceConfig).PodNamespace
```

Comment:
why is there a "v" prefix for namespace only?

Reply:
There used to be other var named podNS so I added "v" to avoid conflict. Now it is ok to name it "podNS".